.idea/workspace.xml

```diff
@@ -5,24 +5,14 @@
   </component>
```

Comment:
We should gitignore this file, there're a lot of personal preferences in this file.